<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ODEClass</class>
 <widget class="QMainWindow" name="ODEClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1129</width>
    <height>762</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ODE</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>40</y>
      <width>291</width>
      <height>561</height>
     </rect>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QDoubleSpinBox" name="ballVxBox">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>110</y>
        <width>62</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maximum">
       <double>360.990000000000009</double>
      </property>
      <property name="value">
       <double>10.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="ballVyBox">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>110</y>
        <width>62</width>
        <height>25</height>
       </rect>
      </property>
      <property name="value">
       <double>30.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>101</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Gun Barrel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>61</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Azimuth</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>110</y>
        <width>81</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Elevation</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>211</width>
        <height>171</height>
       </rect>
      </property>
      <property name="title">
       <string>Integral Method</string>
      </property>
      <widget class="QRadioButton" name="ballEular">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>119</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Eular</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="ballMidPoint">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>119</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Mid Point</string>
       </property>
      </widget>
     </widget>
     <widget class="QSpinBox" name="gunHeight">
      <property name="geometry">
       <rect>
        <x>83</x>
        <y>60</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>66</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Height</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="ballMassBox">
      <property name="geometry">
       <rect>
        <x>83</x>
        <y>200</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
      <property name="value">
       <number>1000</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>66</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Projectile</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>200</y>
        <width>66</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Mass</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>66</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Force</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="ballPowerAmountMass">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>260</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="value">
       <number>3</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>260</y>
        <width>121</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Powder Amount</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ballStopBtn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>490</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ballStartBtn">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>490</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Play</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1129</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ode.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>ballStartBtn</sender>
   <signal>clicked()</signal>
   <receiver>ODEClass</receiver>
   <slot>BallStartPlay()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>995</x>
     <y>630</y>
    </hint>
    <hint type="destinationlabel">
     <x>962</x>
     <y>660</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ballStopBtn</sender>
   <signal>clicked()</signal>
   <receiver>ODEClass</receiver>
   <slot>BallStopPlay()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>850</x>
     <y>632</y>
    </hint>
    <hint type="destinationlabel">
     <x>742</x>
     <y>643</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ballEular</sender>
   <signal>clicked()</signal>
   <receiver>ODEClass</receiver>
   <slot>BallSetEularMethod()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>886</x>
     <y>471</y>
    </hint>
    <hint type="destinationlabel">
     <x>748</x>
     <y>470</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ballMidPoint</sender>
   <signal>clicked()</signal>
   <receiver>ODEClass</receiver>
   <slot>BallSetMidPointMethod()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>851</x>
     <y>507</y>
    </hint>
    <hint type="destinationlabel">
     <x>749</x>
     <y>506</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>BallStartPlay()</slot>
  <slot>BallStopPlay()</slot>
  <slot>BallSetEularMethod()</slot>
  <slot>BallSetMidPointMethod()</slot>
 </slots>
</ui>
